CMakeLists.txt

```diff
@@ -245,18 +245,25 @@ if (BENCHMARK_USE_LIBCXX)
   endif()
 endif(BENCHMARK_USE_LIBCXX)
 
+set(EXTRA_CXX_FLAGS "")
+if (WIN32 AND "${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang")
+  # Clang on Windows fails to compile the regex feature check under C++11
+  set(EXTRA_CXX_FLAGS "-DCMAKE_CXX_STANDARD=14")
```

Comment:
Could you please check, is it still broken with `set(EXTRA_CXX_FLAGS "-DCMAKE_CXX_STANDARD=11")` ?

Reply:
i expect it will as per the original bug #974.

Reply:
@dominichamon is correct

@LebedevRI 14 and up is required for the patch to fix Clang on Windows. 11 is already the default of `try_run` and `try_compile`